_maps/tgstation2.dm

```diff
@@ -15,6 +15,8 @@ z7 = empty space
 
 #if !defined(MAP_FILE)
 
+		#define TITLESCREEN "title" //Add an image in misc/fullscreen.dmi, and set this define to the icon_state, to set a custom titlescreen for your map
```

Comment:
Is it actually required? There is a check in New() to make sure it is defined before it attempts to set it